run_tests.py

```diff
@@ -111,6 +112,8 @@ def test_seed(seed, sim='qemu', repeat=1, retry=0):
     if args.ovpsim:
         sim = 'ovpsim'
 
+    # Build initial ramdisk before running tests
+    subprocess.call('./build_cpio.sh')
```

Comment:
Are you certain this is what you wanted to do? This way the preparation of the ramdisk isn't a part of the build process, but becomes a testing feature instead (...?). It will regenerate the ramdisk every time the test runner script is used, even if no changes were done. It will **not** regenerate the ramdisk when launching the kernel normally (`./launch ...`). I suppose the `./build_cpio.sh` script should be referenced in the main `Makefile`, so that it becomes a part of the build process, ramdisk image gets regenerated on demand etc.